README.md

```diff
@@ -30,12 +30,14 @@ I summarize pretrained BERT models for Japanese by `word segmentation algorithm`
 | akirakubo (b)              | SudachiPy (SudachiDict_core + A mode) for Wikipedia and Aozora bunko written in `新仮名` + MeCab (unidic_qkana) for Aozora bunko written in `旧仮名` | WordPiece                                 | subword-nmt (BPE)                                                      |
 | The University of Tokyo    | MeCab (mecab-ipadic-neologd + user dic (J-MeDic)                                                                                                     | WordPiece                                 | ? (BPE)                                                                |
 | Laboro.AI Inc.             | --                                                                                                                                                   | Sentencepiece (without word segmentation) | Sentencepiece (model_type=unigram)                                     |
+| Bandai Namco Research Inc. | MeCab (mecab-ipadic)                                                                                                                                 | WordPiece                                 | Sentencepiece (model_type=bpe)                                         |
```

Comment:
## DistilBERT

https://github.com/BandaiNamcoResearchInc/DistilBERT-base-jp/blob/master/docs/GUIDE.md#%E4%BA%8B%E5%89%8D%E5%AD%A6%E7%BF%92

In DistilBERT, `bert-base-japanese-whole-word-masking` is used.

## BERT by Tohoku Univ

1. `huggingface/transformers`

<img width="2078" alt="Screen Shot 2020-08-05 at 21 33 21" src="https://user-images.githubusercontent.com/5164000/89413759-4efb1280-d764-11ea-98d1-81be6b3cb350.png">

They published `bert-japanese` and `bert-japanese-char` (without considering masking).

- https://github.com/huggingface/transformers/pull/1839#issuecomment-555841527

2. `cl-tohoku/bert-japanese`

<img width="856" alt="Screen Shot 2020-08-05 at 21 42 27" src="https://user-images.githubusercontent.com/5164000/89413946-97b2cb80-d764-11ea-9b14-8e0dd585299e.png">

- https://github.com/cl-tohoku/bert-japanese#pretrained-models

`bert-base-japanese-whole-word-masking` should be BERT with BPE.
(If a model uses a character-based vocabulary, its name should include `-char`.)